lib/linguist/languages.yml

```diff
@@ -49,6 +49,16 @@ ABAP:
   - ".abap"
   ace_mode: abap
   language_id: 1
+ABNF:
+  type: data
+  ace_mode: text
+  extensions:
+  - ".abnf"
+  interpreters:
+  - ex_abnf
```

Comment:
This field is for languages that can be identified from interpreter directives. =)

For instance, a file with `#!/usr/bin/env node` in its first line can be identified as JavaScript by listing `node` as an interpreter.

Since that doesn't apply here, it's safe to leave this out.